package.json

```diff
@@ -4,7 +4,6 @@
   "description": "Rewriter presubmit script for get-originals work",
   "main": "lib/index.js",
```

Comment:
This is not a big deal since we're not really (yet??) creating a distributable npm package, but the best practice here would be to make the `main` lib/rewriter.js, as the main "programmatic" entry point people would get from `require("get-originals-rewriter")`, and then add a `"bin": "bin/index.js"`, as the executable.